Geo-Cordinate Locator/Requirements.txt

```diff
@@ -0,0 +1,8 @@
+The packages used are:
+1) tkinter
+2) geopy
+
+If geopy is not installed, open command prompt and use the following command:
+pip install geopy
+I hope it resolves any kind of problems that may arise.
+Also make sure your machine is connected to Internet while you are executing this program.
```

Comment:
Your don't write requirements like this..
1. Change **Requirements.txt** to **requirements.txt**
2. Here's how you write **requirements.txt**
```
tkinter==8.6.10
geopy==2.0.0
```
So, you can install them all the packages used at once `pip install -r requirements.txt`

Reply:
check if its alright or not.